core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -383,6 +392,27 @@ var TopicsAndSkillsDashboardPage = function() {
     await waitFor.pageToFullyLoad();
   };
 
+  this.unassignSkillWithIndex = async function(index, topicIndex) {
```

Comment:
I know I harp on this all the time, but any chance we could identify topics and skills by their names instead of index?

Reply:
mmm, Made the skill to be identified by name, for the topic, I am not that sure about topics, Since it is a radio button.

Reply:
That's fine. Thanks!